src/resources/triton_python_backend_utils.py

```diff
@@ -542,3 +542,4 @@ def add_output(self, output):
 TRITONSERVER_REQUEST_FLAG_SEQUENCE_START = 1
 TRITONSERVER_REQUEST_FLAG_SEQUENCE_END = 2
 TRITONSERVER_RESPONSE_COMPLETE_FINAL = 1
+TRITONSERVER_REQUEST_RELEASE_RESCHEDULE = 2
```

Comment:
does it make sense to add release_all to be complete?

Reply:
Didn't expose the `release_all` flag since it's the default behavior, but I think we can add it just to make it complete.

Reply:
Added release_all flag.